cmd/progressbar/update.go

```diff
@@ -71,12 +81,20 @@ func NewUpdate(name string, change int) BarUpdate {
 	}
 }
 
-func NewDynamicUpdate(name string, change int, totalChange int64) DynamicBarUpdate {
+func NewDynamicUpdate(name string, change int) DynamicBarUpdate {
 	return DynamicBarUpdate{
 		BarUpdate: BarUpdate{
 			BarName: name,
 			Change:  change,
 		},
+	}
+}
+
+func NewDynamicTotalUpdate(name string, totalChange int64) DynamicBarUpdate {
```

Comment:
never used

Reply:
for future use. this is a needed functionality for dynamic bars

Reply:
like comment above - please explain what it does then